### ISSUE TITLE
Chapter is not highlighted video in latest Version 24.5.0

### ISSUE THREAD

**Issue by bhavtosh** [smplayer-dev/smplayer#1015]
Kindly refer to attached photo & fix this BUG. 

If a video has chapters then SM player does not highlight the current chapter being played.

Hope to see it fixed in next release including beta.

I've been using this amazing video player since ver 21.10

![sm player BUG](https://github.com/user-attachments/assets/89512dd2-c196-4262-9eac-1f95b6f75303)

**Comment by smplayer-dev** [comment]
I don't think this is a bug. The chapters are timestamps marking the beginning of each chapter. When you select a chapter, SMPlayer will jump to that point.

**Comment by bhavtosh** [comment]
Hello, it seems to be a BUG. 
I opened the same video file with chapters & VLC highighted the 1st chapter on its own. And it highlights the chapter that i choose.

Kindly see if this useful feature can be added to the next build of SM player. Good day.

![Image 1](https://github.com/user-attachments/assets/7778f9f0-64ca-451e-b68e-be7b8a8308ce)

![Image 2](https://github.com/user-attachments/assets/9679d949-5240-48bf-885a-54e581483200)

**Comment by bhavtosh** [comment]
> Hello, it seems to be a BUG. I opened the same video file with chapters & VLC highighted the 1st chapter on its own. And it highlights the chapter that i choose.
> 
> Kindly see if this useful feature can be added to the next build of SM player. Good day.

Hello, it seems to be a BUG.
I opened the same video file with chapters & VLC highighted the 1st chapter on its own. And it highlights the chapter that i choose.

Kindly see if this useful feature can be added to the next build of SM player. Good day.

![Image 1](https://github.com/user-attachments/assets/40eff3b2-a63f-4794-860d-ed064dd2d30f)
![Image 2](https://github.com/user-attachments/assets/922ca97c-0f96-4872-91e4-b3d179707ec8)

**Comment by bhavtosh** [comment]
Waiting for comments from developer...
